.cloudgov/manifest.yml

```diff
@@ -14,7 +14,7 @@ applications:
         command: yarn start-workers
         disk_quota: 2G
         instances: 1
-        memory: ((memory))
```

Comment:
Any idea what it was getting here before?

Reply:
Any double parentheses `((...))` in a CF manifest file references a variable.  See [here](https://github.com/18F/federalist/blob/staging/.cloudgov/vars/production.yml) for our production deploys and [here](https://docs.cloudfoundry.org/devguide/deploy-apps/manifest-attributes.html#variable-substitution) for the CF docs.

Reply:
Yes, I could tell it as going to be a variable. I was curious what the value was before if not 2GB. (And for that matter couldn't we also increase it by editing the variable instead of the code?)

Reply:
(I did find the answer to my production environment value in the production deploys.